projects/gloo/api/v1/enterprise/options/ai/ai.proto

```diff
@@ -208,6 +208,66 @@ message UpstreamSpec {
     string model = 4;
   }
 
+  enum SigningAlgorithm {
+    // Use SigV4 for signing
+    AWS_SIGV4 = 0;
+
+    // Use SigV4A for signing
+    AWS_SIGV4A = 1;
+  }
+
+  // Settings for the Bedrock LLM provider
+  message Bedrock {
+    // The authorization config used to access authenticated AWS Bedrock services.
+    AwsCredentialProvider credential_provider = 1;
+    // Optional: Send requests to a custom host and port, such as to proxy the request,
+    // or to use a different backend that is API-compliant with the upstream version.
+    // Note: For AWS Bedrock, if custom_host is set, host_rewrite will be used to override the Host header before signing the request
+    CustomHost custom_host = 2;
+    // Optional: Sets the model-id name.
+    // If unset, the model name is taken from the request.
+    string model = 3;
+    // Required: region string.
+    //
+    // When signing_algorithm is set to ``AWS_SIGV4`` the region is a standard AWS `region <https://docs.aws.amazon.com/general/latest/gr/rande.html>`_ string for the service
+    // hosting the HTTP endpoint.
+    //
+    // Example: us-west-2
+    //
+    // NOTE: multiple regions are not currently supported
```

Comment:
Is there an implication for this? such as all the HTTP endpoints must then be in the same region?
```suggestion
    // NOTE: Multiple regions are not currently supported.
```

Reply:
Users can't set the string to "us-west-1,us-east-1" currently, but that would be something we want to support in the future.

Reply:
Updated in: https://github.com/solo-io/gloo/pull/10796/commits/3c939a50f339e787c8010585b1b55bcafc319ecb